backend/columnar.js

```diff
@@ -2091,9 +2091,7 @@ class BackendDoc {
     docState.lastIndex[ops.objId] = visibleCount
     for (let col of docState.opsCols) col.decoder.reset()
 
-    let outCols = docState.allCols.map(columnId => {
-      return {columnId, encoder: encoderByColumnId(columnId)}
-    })
+    let outCols = docState.allCols.map(columnId => ({columnId, encoder: encoderByColumnId(columnId)}))
```

Comment:
this comes up a few times -- if you have an arrow function returning an object you can just wrap it in parentheses instead of creating a block and returning the value